client/app/assets/less/ant.less

```diff
@@ -47,12 +47,6 @@
 @zindex-picker: 2050;
 @zindex-tooltip: 2060;
 
-.@{drawer-prefix-cls} {
-  &.help-drawer {
-    z-index: 3000; // help drawer should be topmost
```

Comment:
This didn't allow a tooltip to appear for drawer elements.
I tested and found no regression.

Reply:
Proof:

<img width="1005" alt="Screen Shot 2019-06-08 at 12 46 11" src="https://user-images.githubusercontent.com/486954/59145359-7123c400-89eb-11e9-9ec8-c9ab8efe7fd1.png">